### DISCUSSION TITLE
default table length and width are automatically generated

### DISCUSSION THREAD

**Discussion by graczhual** [pradyunsg/furo#379] [Potential Issue] [answered]
When we use autosummary of sphinx, the automatically generated table doesn't look good.
<img width="978" alt="image" src="https://user-images.githubusercontent.com/87636820/154921357-10c60207-2a3e-4f63-ac0b-4460f33fe2e3.png">
https://graviti-python-sdk--18.org.readthedocs.build/en/18/reference/api/generated/graviti.DataFrame.html
Is there any way to modify the presentation of the table, such as cancel auto center, customize the length and width, etc.

**Comment by graczhual** [comment]
？

**Comment by pradyunsg** [comment, marked as answer]
> Is there any way to modify the presentation of the table, such as cancel auto center, customize the length and width, etc.

Not as far as I'm aware, no.

What do you want to change here?

**Reply by pradyunsg** [reply]
Marking this as an answer, since OP didn't respond.
